src/netius/test/extra/proxy_r.py

```
@@ -1398,3 +1464,230 @@ def _request(self, path, method="GET", headers=None, body=None):
+
+    SMALL = b"ab"
+
+    JPEG = b"\xff\xd8" + b"j" * 4000
+
+    ENCODED = zlib.compress(b"netius " * 900)
+
+    BROTLI = b"\x1b\x2e\x00\x00" + b"b" * 200
+
+    @classmethod
+    def setUpClass(cls):
+        if http_client == None:
```

Ruff (0.16.1)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)